docs/setup/native.md

```diff
@@ -63,14 +63,18 @@ the `vets-api` directory, run `./bin/setup-osx && source ~/.bash_profile && cd
 1. Install overcommit `overcommit --install --sign`
 1. Setup localhost certificates / keys:
    - Create certs directory within config:  `mkdir ./config/certs`
-   - Copy the [certificate][certificate] to `./config/certs/vetsgov-localhost.crt`
-   - Copy the [key][key] to `./config/certs/vetsgov-localhost.key`
-   - *NOTE*: If you don't have access to these keys, running the following
-     commands will provide basic functionality, such as for running unit tests:
-   - `touch ./config/certs/vetsgov-localhost.crt`
-   - `touch ./config/certs/vetsgov-localhost.key`
+   - *NOTE*: If you don't have access to these keys, [Create your own self-signed certificate](https://github.com/department-of-veterans-affairs/vets.gov-team/tree/master/Products/Identity/Login/IDme/development-certificates#creating-your-own-self-signed-certificates)
+   - Copy the [certificate](https://github.com/department-of-veterans-affairs/vets.gov-team/blob/master/Products/Identity/Login/IDme/development-certificates/vetsgov-localhost.crt) to `./config/certs/vetsgov-localhost.crt`
+   - Copy the [key](https://github.com/department-of-veterans-affairs/vets.gov-team/blob/master/Products/Identity/Login/IDme/development-certificates/vetsgov-localhost.key) to `./config/certs/vetsgov-localhost.key`
+
```

Comment:
All of these links point to the `vets.gov-team` repo, which is private. The problem is that `vets-api` rails server wont even start unless these files exist. So for peeps w/o access to `vets.gov-team`, it's important for them to understand that empty files will at least allow them to startup the server.

Reply:
Plus they're either moving to `va.gov-team` or the sensitive repo.